### ISSUE TITLE
Fix master deployment

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden-services#284]
## Problem Definition

Currently the continuous deployment doesn't work.

**Comment by palango** [comment]
After #271 and #293 the builder still fails.

```
builder_1       | Switching to /root/raiden-services
builder_1       | Container names = ['pfs-ropsten', 'pfs-rinkeby', 'pfs-kovan', 'pfs-goerli', 'ms-ropsten', 'ms-rinkeby', 'ms-kovan', 'ms-goerli', 'msrc-ropsten', 'msrc-rinkeby', 'msrc-kovan', 'msrc-goerli']
builder_1       | [Errno 2] No such file or directory: '/root/raiden-services'
```

**Comment by palango** [comment]
Took some iterations, but fixed as of #306
